scripts/pre-push

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+source ${0%/*}/utils/message
+
+title "Run pre-push hook..."
+
+# Run the complete suite of tests
+npm run test -- -b
```

Comment:
We should actually run the linting as well to ensure that we introduce errors with a rebase...